wire-ios-data-model/WireDataModel.xcodeproj/project.pbxproj

```diff
@@ -2738,7 +2722,6 @@
 			isa = PBXGroup;
 			children = (
 				597A3BB52B6418090020E337 /* Availability */,
-				597A3BB82B641EA20020E337 /* UserStatus */,
```

Comment:
Why is this removed? Was it a duplicate reference?

Reply:
Yes it was an empty lost reference in the project manager, probably because of a merge conflict.